tests/tests/clients-1-test.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env roundup
+#
+#/ usage:  rerun stubbs:test -m tests -p clients [--answers <>]
+#
+
+# Helpers
+# -------
+[[ -f ./functions.sh ]] && . ./functions.sh
+
+# The Plan
+# --------
+describe "clients"
+
+source ../lib/clients.sh
+
+TEST_ROOT="$(mktemp -d /tmp/roundup-test.XXX)"
+
+function cleanup {
+  rm -r "${TEST_ROOT}"
+}
+
+trap cleanup EXIT
+
+it_sets_up_released_deisctl() {
+
+  function download-client {
+    [ ${1} == "deisctl" ] &&
+    [ ${2} == "1.8.0" ]
```

Comment:
This is a test, so the line below

`setup-deisctl-client "1.8.0"`

eventually I want to see download the 1.8.0 deisctl client (which I intercept in the inner `download-client` function here).